courses/templates/partials/metadata-tiles.html

```diff
@@ -5,7 +5,7 @@
         <li>
           <span class="title">START DATE</span>
           <span class="text">{{ course.next_run_date|date:"F j, Y" }}</span>
-          <span class="dates-link"><a href="#">More Dates</a></span>  {# TODO Need to do in separate story#}
+          <span class="dates-link"><a class="dates-tooltip" id="datesPopover" data-trigger="focus" href="#" onClick="event.preventDefault();" data-toggle="popover" data-html="true" data-placement="auto" title="More dates available for this course" data-content="<div><p>Click below to enroll in one of these dates</p></div>{% for date in course.unexpired_run_dates %}<div><a class='date-link' href='#'>Start Date {{ date|date:'F j, Y' }}</a></div>{% endfor %}">More Dates</a></span>
```

Comment:
plz break the `<a>` or `<span>` into multi lines

Reply:
Done